phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexScrutinyMapper.java

```diff
@@ -228,8 +256,17 @@ private void processBatch(Context context)
                     buildTargetStatement(targetStatement);
 
             // fetch results from the target table and output invalid rows
+            if (isTest) {
+                // change the current time past ttl
+                ScrutinyTestClock clock = new ScrutinyTestClock(3600000);
```

Comment:
How does the isTest guarantee that TTL is 360000? Perhpas make a const that is used here as well as in the test that creates table with this TTL.

Reply:
Looks like Line 94 in the file above.

Reply:
I ll extract it to a constant.